### ISSUE TITLE
Failure to run Laravel Dusk

### ISSUE THREAD

**Issue by JenniferZhou1999** [laravel/dusk#543]
I just downloaded Laravel Dusk and when I typed php artisan dusk in my project folder, I received this error. 

1) Tests\Browser\ExampleTest::testBasicExample
Did not see expected text [Laravel] within element [body].
Failed asserting that false is true.

/Users/jenniferzhou/Documents/Cypress/MarketPlace/vendor/laravel/dusk/src/Concerns/MakesAssertions.php:348
/Users/jenniferzhou/Documents/Cypress/MarketPlace/vendor/laravel/dusk/src/Concerns/MakesAssertions.php:319
/Users/jenniferzhou/Documents/Cypress/MarketPlace/tests/Browser/ExampleTest.php:20
/Users/jenniferzhou/Documents/Cypress/MarketPlace/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:67
/Users/jenniferzhou/Documents/Cypress/MarketPlace/tests/Browser/ExampleTest.php:21

FAILURES!
Tests: 1, Assertions: 1, Failures: 1.

I have tried everything I saw online, but cannot seem to fix my problem. 

**This is my DuskTestCase.php file:** 

`<?php

namespace Tests;

use Laravel\Dusk\TestCase as BaseTestCase;
use Facebook\WebDriver\Chrome\ChromeOptions;
use Facebook\WebDriver\Remote\RemoteWebDriver;
use Facebook\WebDriver\Remote\DesiredCapabilities;

abstract class DuskTestCase extends BaseTestCase
{
    use CreatesApplication;

    /**
     * Prepare for Dusk test execution.
     *
     * @beforeClass
     * @return void
     */
    public static function prepare()
    {
        static::startChromeDriver();
    }

    /**
     * Create the RemoteWebDriver instance.
     *
     * @return \Facebook\WebDriver\Remote\RemoteWebDriver
     */
    protected function driver()
    {
        $options = (new ChromeOptions)->addArguments([
            '--disable-gpu',
            '--headless'
        ]);

        return RemoteWebDriver::create(
            'http://localhost:9515', DesiredCapabilities::chrome()->setCapability(
                ChromeOptions::CAPABILITY, $options
            )
        );
    }
}
`

**This is my .env file**

`APP_NAME=Laravel
APP_ENV=local
APP_KEY=base64:aXlPbKtuLqJ8sRrr2i7gEyUlWWvoaED/iYdVTTujsS8=
APP_DEBUG=true
APP_LOG_LEVEL=debug
APP_URL=http://market.test

DB_CONNECTION=mysql
DB_HOST=market.test
DB_PORT=3306
DB_DATABASE=market
DB_USERNAME=homestead
DB_PASSWORD=secret

BROADCAST_DRIVER=log
CACHE_DRIVER=file
SESSION_DRIVER=file
SESSION_LIFETIME=120
QUEUE_DRIVER=sync

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_DRIVER=smtp
MAIL_HOST=smtp.mailtrap.io
MAIL_PORT=2525
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_APP_CLUSTER=mt1
`

Based on what I have read, it may be a problem with the APP_URL in my .env file. However, I do not want to change that, because I want to be able to access my project by typing in market.test in the browser. Is there any other way to fix this issue? Thank you!

**Comment by staudenmeir** [comment]
Does your `/` page actually contain "Laravel"? Does it work when you use `APP_URL=http://localhost`?

**Comment by JenniferZhou1999** [comment]
I changed APP_URL to APP_URL=http://localhost and then typed in php artisan dusk. However, I still get the same error.

**Comment by staudenmeir** [comment]
What does the screenshot in `tests/Browser/screenshots` show?

**Comment by JenniferZhou1999** [comment]
My tests/Browser/screenshots/failure-testBasicExample-0.png file only shows a blank white space.

**Comment by staudenmeir** [comment]
Are you running `php artisan dusk` in Homestead (via SSH) or on your host machine?

**Comment by JenniferZhou1999** [comment]
I always cd into my MarketPlace project and then run php artisan dusk (I do not run php artisan dusk in my homestead folder), so I believe my host machine.

**Comment by staudenmeir** [comment]
`http://market.test` works in your browser? Is the screenshot also blank when you run Dusk with the original  `APP_URL=http://market.test`?

**Comment by JenniferZhou1999** [comment]
http://market.test does work in my browser(my website shows up as it should). When I run php artisan dusk with the original APP_URL=http://market.test, a screenshot of the home page of my website shows up in the file.

**Comment by staudenmeir** [comment]
Does your home page actually contain the word "Laravel"? The example test is meant to work with the default [welcome page](https://github.com/laravel/laravel/blob/master/resources/views/welcome.blade.php).

**Comment by JenniferZhou1999** [comment]
My home page no longer contains the word "Laravel". I changed the words in the title tag to Cypress. (I have included my welcome.blade.php below). Does that mean the reason the test is outputting error, is because I was using a test that is testing a file that does not exist? 

`

<!DOCTYPE html>
<html>
<head>




</head>

<!doctype html>
<html lang="{{ app()->getLocale() }}">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Cypress</title>

        
        <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
        <title>Cypress</title>



        
        {{--<script async src="https://www.googletagmanager.com/gtag/js?id=UA-112480175-1"></script>--}}
        {{--<script>--}}
            {{--window.dataLayer = window.dataLayer || [];--}}
            {{--function gtag() {--}}
                {{--dataLayer.push(arguments);--}}
            {{--}--}}
            {{--gtag('js', new Date());--}}

            {{--gtag('config', 'UA-112480175-1');--}}
        {{--</script>--}}


        
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>


        <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
        <link href="css/land.css" rel="stylesheet" type="text/css"/>
        <link href="css/w3.css" rel="stylesheet" type="text/css"/>
        <style>
            ul.dropdown-menu {
                position: static;
                float: none;
                width: 500px;
            }
            a {
                color: black;
                text-decoration: none;
            }

        </style>
    </head>
    <body>
    <div >
        <div class="_33252b-page _33252b-white">

            <div class="_a3c844-wrapper">

                
                <div class="_660a47-header _660a47-absolute">
                    
                    <div class="_660a47-logo">

                        <a href="{{ url('/') }}">
                            <img src="/img/Logo.png"
                                 style="width: 170px; height: 70px; float: left; background-color: transparent">
                        </a>
                        <label style="font-family: 'Abadi MT Condensed Extra Bold'; color: darkorange"><strong>Beta</strong></label>



                    </div>

                    
                    <div class="_660a47-rightBlock">
                        <div class="_334396-button _660a47-authButton" >

                            <div class="_334396-userName _334396-newHeader">

                                @if (Auth::guard('cuser')->guest()&&Auth::guard('admin')->guest()&&Auth::guest())


                                    {{--<a class="nav-link w-nav-link" href="{{ url('/insight/pubhome') }}"--}}
                                    {{--style="margin-right: 20px" title="Explore sample insights and sign up for more!">Explore</a>--}}
                                    <a class="nav-link w-nav-link" href="{{ url('/login') }}"
                                       style="margin-right: 20px">Expert Login</a>

                                    <a class="nav-link w-nav-link" href="{{ url('/company/login') }}"
                                       style="margin-right: 20px">Company Login</a>



                                    <a class="nav-link w-nav-link" href="{{ url('/signup') }}"style="margin-right: 20px">Join the Waitlist</a>



                                @elseif(Auth::guard('cuser')->user())
                                    <div class="nav-link w-nav-link">
                                        <div class="w3-dropdown-hover"
                                             style="color:#58595b; background: transparent; font-size: 15px">
                                            <img src="/uploads/avatar/{{Auth::guard('cuser')->user()->avatar}}" alt=""
                                                 class="img-rounded"
                                                 style="object-fit: cover;width: 32px; height: 32px; margin-bottom: 5px">
                                            {{ Auth::guard('cuser')->user()->company_name }}<span class="caret"></span>
                                            <div class="w3-dropdown-content w3-bar-block w3-border">
                                                <a href="/company/home" class="w3-bar-item w3-button"> Home</a>

                                                <a href="/company/{{\Auth::guard('cuser')->user()->id}}"class="w3-bar-item w3-button"> Company Profile</a>

                                                <a href="/company/{{\Auth::guard('cuser')->user()->id}}/setting"class="w3-bar-item w3-button"> Edit My Company Profile</a>

                                                <a href="{{ url('/company/logout') }}" class="w3-bar-item w3-button">Logout</a>
                                            </div>
                                        </div>
                                    </div>


                                @else
                                    <div class="nav-link w-nav-link">
                                        <div class="w3-dropdown-hover"
                                             style="color:#58595b; background: transparent; font-size: 15px">
                                            <img src="/uploads/avatar/{{\Auth::user()->avatar}}" alt=""
                                                 class="img-rounded"
                                                 style="object-fit: cover;width: 32px; height: 32px; margin-bottom: 5px">
                                            {{ Auth::user()->name }}<span class="caret"></span>
                                            <div class="w3-dropdown-content w3-bar-block w3-border">
                                                <a href="/home" class="w3-bar-item w3-button"> Home</a>
                                                <a href="/user/{{\Auth::id()}}" class="w3-bar-item w3-button"> My
                                                    Page</a>
                                                <a href="/user/me/setting" class="w3-bar-item w3-button"> Edit My
                                                    Profile</a>
                                                <a href="{{ url('/logout') }}" class="w3-bar-item w3-button">Logout</a>
                                            </div>
                                        </div>
                                    </div>

                                @endif

                            </div>
                        </div>
                    </div>

                </div>


                <div class="_b4ffb0-container">
                    <div class="_b4ffb0-textWrapper">
                        <div>
                            {{--<div class="_a49fa3-title" style="width: 700px; margin-left: 12px;">--}}
                            {{--The destination for K-12 educators to collaborate, grow, and succeed--}}
                            {{----}}
                            {{--</div>--}}
                            <div class="_a49fa3-title" style="width: 700px; margin-left: 12px;">
                                Cypress

                            </div>
                            <h3 class="_a49fa3-subtitle">
                                Currently in private beta </h3></div>
                        {{--<h3 class="_a49fa3-subtitle">--}}
                        {{--Data-driven, collaborative, career management and professional learning that integrates--}}
                        {{--seamlessly with your workflow.</h3></div>--}}

                        <br>


                        <div class="_b4ffb0-buttonWrapper" style="margin-top: 10px">
                            <div class="_4c31bd-basicButton  _4c31bd-shapeRound _4c31bd-sizeLarge"
                                 style="background-color: #a5d5a7; margin-left: 50px; margin-top: 10px">
                                <a href="{{ url('/login') }}" style="color: black" title="Login here if you are a teacher">
                                    Expert Login</a>
                            </div>
                            <div class="_4c31bd-basicButton  _4c31bd-shapeRound _4c31bd-sizeLarge"
                                 style="background-color: #a5d5a7; margin-top: 10px; margin-bottom: 10px; margin-left: 40px">
                                <a href="{{ url('/company/login') }}" style="color: black"
                                   title="Sign up here for access to our private Beta">Company Login</a>
                            </div>

                            <hr>
                            <div class="_4c31bd-basicButton  _4c31bd-shapeRound _4c31bd-sizeLarge"
                                 style="background-color: #f9ac63; margin-left: 40px;margin-top: 20px">
                                <a href="{{ url('/signup') }}" style="color: black"
                                   title="Login here if you are a Cypress Insights Contributor and not a teacher">
                                    Join the Waitlist </a>
                            </div>


                        </div>

                    </div>
                </div>

               
                        
                <footer class="_fbd2f5-wrap">
                    <div class="_fbd2f5-copyright _fbd2f5-clearfix _fbd2f5-body">
                        Copyright &copy; Cypress Ed, Inc. 2018. All Rights Reserved
                        <br>
                        <br>

                        <a href="{{ route('termsconditions') }}">Terms of Use </a> |<a href ="{{ route('privacypolicy') }}"> Privacy Policy</a>
                    </div>
                </footer>
            </div>

        </div>
    </div>

        
    </body>
</html>




`

**Comment by staudenmeir** [comment]
> Did not see expected text [Laravel] within element [body].

As the error message says, the test fails because the given text doesn't appear on the page.

You have to change it to something like `->assertSee('Cypress')`.

**Comment by JenniferZhou1999** [comment]
Thank you so much! I do not know how I did not figure that out! Thank you!
